src/julia_threads.h

```diff
@@ -211,7 +211,7 @@ typedef struct _jl_tls_states_t {
     int16_t tid;
     int8_t threadpoolid;
     uint64_t rngseed;
-    volatile size_t *safepoint;
+    _Atomic(volatile size_t *) safepoint; // may be changed to the suspend page by any thread
```

Comment:
This is more of a dumb question than a review, but what is the difference here from volatile and `_Atomic`, and why do we need both